tests/test_crawl.py

```diff
@@ -251,6 +251,19 @@ def start_requests(self):
     def test_open_spider_error_on_faulty_pipeline(self):
         settings = {
             "ITEM_PIPELINES": {
+                "tests.pipelines.LegacyZeroDivisionErrorPipeline": 300,
```

Comment:
This was intentional, to test the legacy way of defining an item pipeline. The test method naming might have confused you, but I've fixed this